pages/index.js

```diff
@@ -1,3 +1,28 @@
-export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+import ArticleList from "../components/ArticleList";
+import styled from "styled-components";
+import useSWR from "swr";
+import { fetcher } from "../helpers/api";
+//import articlesDb from "../helpers/db.json";
+
+function Home() {
+  const { data: articles, error } = useSWR("/api/articles", fetcher);
+  console.log(articles);
+
+  if (!articles) {
+    return <h4>loading...</h4>;
+  }
+  return (
+    <>
+      <h2>Alle Artikel in deiner Nähe:</h2>
+      <UL>
+        <ArticleList articles={articles} />
+      </UL>
+    </>
+  );
```

Comment:
I suggest to move the `<h2>` into the `<ArticleList>` (see comments in ArticleList components). 
Apart from the fact that `<UL>` is a very bad name, you no longer need this element. 
Now it's super clean 🧽

```suggestion
  return (
        <ArticleList articles={articles} />
  );
```

Reply:
thanks for your very good feedback. I fixed it.